jupyterlab_gcpscheduler/src/components/scheduler_form.tsx

```diff
@@ -127,11 +126,11 @@ function getName(notebookName: string) {
 
 const localStyles = stylesheet({
   scroll: {
-    maxHeight: '52vh',
+    maxHeight: '56vh',
     overflowY: 'scroll',
     overflowX: 'hidden',
-    borderTop: '0px solid #777',
-    borderBottom: '0px solid #777',
+    borderTop: '1px solid #DADCE0',
```

Comment:
Same as above, should we be trying to use the CSS color variables from https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-light-extension/style/variables.css where possible to avoid theming conflicts?

Reply:
Can do. I was trying to use the color from Figma but I think I found a jp alternative.

Reply:
Excellent. Please resolve once the change is made.